index.html

```diff
@@ -18,7 +18,16 @@
         <h1>Welcome to the student showcase!</h1>
     </div>
     <div class="content">
-        <div>ONE</div>
+        <div>
+            <div id="pic">
+                <img src="https://i.imgur.com/ycdXJOt.jpg" alt="Me" height="200px" width="auto">
+            </div>
+            <div id="words">
+                    <h3>Travis Dellard</h3><br/>
+                    <p id="quote">Last name Ever, first name Greatest.</p><br/>
```

Comment:
You don't need these `<br/>` tags. We'll use `margin`/`padding` in CSS to get the spacing we want.